tests/e2e_tests/integration/07-saml.spec.ts

```diff
@@ -0,0 +1,183 @@
+import axios from 'axios';
+import * as fs from 'fs';
+import * as https from 'https';
+import jwtDecode from 'jwt-decode';
+
+import test, { expect } from '../fixtures/fixtures';
+
+test.describe('SAML Login', () => {
+  test.use({ storageState: 'storage.json' });
+  test.describe('SAML login via sso/id/login', () => {
+    test('Logs in using UI', async ({ environment, context, baseUrl, page, password, username }) => {
+      test.skip(environment !== 'staging');
+      test.setTimeout(240000);
+      const saml_settings={
+        'login':'morty',
+        'password':'panic',
+        'email':'msmith@samltest.id',
+        'idp_url':'https://samltest.id/saml/idp'
+      };
+      const timeout_4s=4000;
+      const timeout_1s=1000;
```

Comment:
since these are constant, they should best be outside the test definition (being strict, they would be a fixture, but since they are unlikely to be reused across this test outside the test is fine).
also: this is JS/TS land, we do camelCase when we define with our things 😛 (or: your PR made me realize, that the linting seems to not be enforced in these tests... 🤯)

Reply:
moved up.